mod/cornercutter/ModFeature/ModSetting/StartingSkillSwapper.cs

```diff
@@ -11,14 +11,14 @@ class StartingSkillSwapper
     {
         static void Postfix(ref DungeonManager __instance)
         {
-            if (!CutterConfig.Instance.CornercutterIsEnabled()) return;
+            CutterConfig cornercutter = CutterConfig.Instance;
+            if (!(cornercutter.CornercutterIsEnabled() && cornercutter.HasCurrentMod)) return;
 
             int currentFloor = __instance.currentFloor + 1;
             WeightedSkill[] startingSkills = CutterConfig.Instance.StartingSkills;

```

Comment:
You can replace `CutterConfig.Instance` with `cornercutter` here.

Reply:
good catch, fixed